.changelog/2258.txt

```diff
@@ -1,3 +1,3 @@
 ```release-note:enhancement
-resource/federated_settings_org_rolemapping: Exposes role_mapping_id
+resource/mongodbatlas_federated_settings_org_rolemapping: Exposes role_mapping_id
```

Comment:
If this was hidden before, suggest "Introduces" or "Adds" instead of "Exposes":
```suggestion
resource/mongodbatlas_federated_settings_org_rolemapping: Introduces role_mapping_id.
```

Reply:
adjusted following convention defined in https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/changelog-process.md#resource-and-provider-enhancements